src/Bicep.Core/PrettyPrintV2/SyntaxLayouts.SyntaxVisitor.cs

```diff
@@ -171,5 +171,7 @@ private void Apply<TSyntax>(TSyntax syntax, DocumentLayoutSpecifier<TSyntax> lay
                 ? TextDocument.From(SyntaxStringifier.Stringify(syntax, this.context.Newline).Trim())
                 : layoutSpecifier(syntax);
         }
+
+        public void VisitAssertDeclarationSyntax(AssertDeclarationSyntax syntax) => this.Apply(syntax, this.LayoutAssertDeclarationSyntax);
```

Comment:
>VisitAssertDeclarationSyntax [](http://example.com/codeflow?start=20&length=28)

nit - Should probably move the method up above `Layout()` on L156 to match existing organization of the file.